README.rst

```diff
@@ -72,7 +72,7 @@ To deploy kebechet on OpenShift cluster. Use the following Ansible command with
     --extra-vars=OCP_TOKEN= <openshift_cluster_token> \
     --extra-vars=KEBECHET_INFRA_NAMESPACE= <openshift_cluster_namespace> \
     --extra-vars=KEBECHET_APPLICATION_NAMESPACE= <openshift_cluster_namespace> \
-    --extra-vars=KEBECHET_CONFIGURATION= <github_repo_config.yaml> \
+    --extra-vars=KEBECHET_CONFIGURATION_PATH= <github_repo_config.yaml> \
```

Comment:
If you want to have this renamed, please change this also in sources so we are consistent - https://github.com/thoth-station/kebechet/blob/master/kebechet/cli.py#L75

Reply:
@fridex @goern IMHO, The variable in https://github.com/thoth-station/kebechet/blob/master/kebechet/cli.py#L75 is dependent on this https://github.com/thoth-station/kebechet/blob/d1e975bc864bb55ae80dbb7511f6cbfff18f2fe6/openshift/cronJob-template.yaml#L46.